### PR TITLE
Bug/testing fix and PSQL data generator

### PR DESCRIPTION
**Pull request by Busyreadingsomething**
[Fix Tests](https://trello.com/c/nZ9WHUzo/7-make-sure-tests-pass-s)
[Data Generators](https://trello.com/c/mFOjHs9Q/9-create-psql-data-generators-s)

### CODE REVIEW

File: SDC_DB/createSchema.js [hunk 1]
```diff
@@ -54,8 +72,8 @@ function write10Million(start = 1e2) {
   }
 
   if (i === 0) {
-    const command = 'mongoimport -d silverspoon -c restaurantMenus --file db/testData.json --numInsertionWorkers 4';
-    exec(command);
+    const command = 'mongoimport -d silverspoon -c restaurantmenus --file db/testData.json --type json --numInsertionWorkers 4';
```
Comment (NgoMike) [inline SDC_DB/createSchema.js:75]: just curious if the flag --type json was required for the command to seed properly?

File: db/dbHelpers.js [hunk 2]
```diff
@@ -58,7 +58,7 @@ const save = (options, cb) => {
 
 const find = (options, cb) => {
   const query = options.query || 'id menu.lunch';
-  const idNum = options.id || 90976;
+  const name = options.name || 'Mr. Gina Armstrong';
```
Comment (NgoMike) [inline db/dbHelpers.js:61]: Mr. Gina Armstrong is a great restaurant

**Review by NgoMike — APPROVED** [review]
I think everything looks good and no changes needed. Just had a few comments. Also, nice updates with the testing!

File: SDC_DB/createSchema.js [hunk 3]
```diff
@@ -30,21 +32,37 @@ const restaurantSchema = mongoose.Schema({
...
+};
+
+function write10Million(start = 1e7) {
   let i = start;
   let freeSpace = true;
 
   while (i > 0 && freeSpace) {
-    const data = generator(i);
+    const data = sampleDataGen(i);
     freeSpace = writeStream.write(`${JSON.stringify(data)}\n`);
```
Comment (MichaelDurfey) [inline SDC_DB/createSchema.js:61]: Is having a new line character necessary with the mongo import method? Just curious. I might have to implement that myself..

**Review by MichaelDurfey — APPROVED** [review]
Looks good

**Pull request merged**